### ISSUE TITLE
Regular ENOENT when using in SSH Remote

### ISSUE THREAD

**Issue by peterbaumert** [monosans/djlint-vscode#323]
When using your extension I am getting regular those errors:

```
2023-07-20 14:10:51.956 [error] Error: Command failed with ENOENT: /var/www/html/DIRECTORY.I.USE/.venv/bin/python -m djlint - --quiet --linter-output-format <filename>{filename}</filename><line>{line}</line><code>{code}</code><message>{message}</message>
spawn /var/www/html/DIRECTORY.I.USE/.venv/bin/python ENOENT
	at Process.ChildProcess._handle.onexit (node:internal/child_process:283:19)
	at onErrorNT (node:internal/child_process:478:16)
	at processTicksAndRejections (node:internal/process/task_queues:83:21)
```

**Comment by monosans** [comment]
I was unable to reproduce the error with:

- VSCode 1.80.1 
- Remote OS: Ubuntu 22.04
- Python 3.11.4
- djLint 1.32.1

How is my environment different from yours?

**Comment by christopherpickering** [comment]
ENOENT normally means the path doesn't exist. Are you sure it is there and your user has access to it?

**Comment by peterbaumert** [comment]
> ENOENT normally means the path doesn't exist. Are you sure it is there and your user has access to it?

it exists and sometimes works, error only pops up like "half" of the time.


> I was unable to reproduce the error with:
> 
> * VSCode 1.80.1
> * Remote OS: Ubuntu 22.04
> * Python 3.11.4
> * djLint 1.32.1
> 
> How is my environment different from yours?

* VSCode 1.80.1
* Remote OS: Debian 11
* Python 3.11.2
* djLint 1.32.0

**Comment by qxpf666** [comment]
1 have the same problems.

**Comment by qxpf666** [comment]
Command failed with ENOENT: e:\BaiduNetdiskWorkspace\python\test\test_env\Scripts\python.exe -m djlint - --quiet --linter-output-format <filename>{filename}</filename><line>{line}</line><code>{code}</code><message>{message}</message> spawn e:\BaiduNetdiskWorkspace\python\test\test_env\Scripts\python.exe ENOENT

**Comment by monosans** [comment]
I have a hunch that ENOENT is not caused by the Python executable, but by the cwd (current working directory) in which the startup occurs.

In <https://github.com/monosans/djlint-vscode/releases/tag/v2023.8.0> I have improved the error output. Now instead of just an error message the entire error object is output, hopefully this will somehow give us a clue as to where to go next in solving this problem.

**Comment by bababash** [comment]
I am having the same problem. The `cwd` definitely looks wrong but I don't know where to change it. 
Using vscode in an ssh tunnel on my server.
Here is my error log:
```
2023-08-06 13:38:05.057 [error] {
	"errno": -2,
	"code": "ENOENT",
	"syscall": "spawn python",
	"path": "python",
	"spawnargs": [
		"-m",
		"djlint",
		"-",
		"--quiet",
		"--linter-output-format",
		"<filename>{filename}</filename><line>{line}</line><code>{code}</code><message>{message}</message>"
	],
	"originalMessage": "spawn python ENOENT",
	"shortMessage": "Command failed with ENOENT: python -m djlint - --quiet --linter-output-format <filename>{filename}</filename><line>{line}</line><code>{code}</code><message>{message}</message>\nspawn python ENOENT",
	"command": "python -m djlint - --quiet --linter-output-format <filename>{filename}</filename><line>{line}</line><code>{code}</code><message>{message}</message>",
	"escapedCommand": "python -m djlint - --quiet --linter-output-format \"<filename>{filename}</filename><line>{line}</line><code>{code}</code><message>{message}</message>\"",
	"stdout": "",
	"stderr": "",
	"cwd": "c:/Users/bababash/AppData/Roaming/Code/User",
	"failed": true,
	"timedOut": false,
	"isCanceled": false,
	"killed": false
```

**Comment by monosans** [comment]
I think I found the cause of the problem. I'll release a fix in half an hour.

**Comment by monosans** [comment]
With v2023.8.3, I've added a temporary fix for this problem. I'm not sure if it works, because I never managed to reproduce the issue.

I also added logs like `Unsupported scheme "{scheme}" for "{fsPath}". Cwd will not be set.`. It would be nice if you could post them here, it would help a lot.

**Comment by peterbaumert** [comment]
Hi thank you for takling this issue.

Here is my output ( only posting one but its always same besides different filenames):

´´´
2023-08-07 12:18:09.289 [warning] Unsupported scheme "dj-embedded-content" for "/file%3A%2F%2F%2Fvar%2Fwww%2Fhtml%2FMYDIR%2Fito%2Fservices%2Ftemplates%2Fservices%2Fvirtual_machine%2Fdetail.html/0.7057242894984939.html". Cwd will not be set.
´´´

**Comment by monosans** [comment]
The issue was caused by using one of these extensions: [Django Template Support](https://marketplace.visualstudio.com/items?itemName=junstyle.vscode-django-support) or [Twig Language](https://marketplace.visualstudio.com/items?itemName=junstyle.twig-language). Both of these extensions create virtual documents that triggered djLint. In v2023.8.6, I made djLint only trigger on documents with the `file` or `vscode-vfs` URI scheme. 

Now the issue should go away.
